src/stdio.jl

```diff
@@ -4,13 +4,34 @@
 
 # logging in verbose mode goes to original stdio streams.  Use macros
 # so that we do not even evaluate the arguments in no-verbose modes
+
+#time() returns utc, now() returns local time
+#FYI this could be wrong by -1 second if the now() call happens just before a second boundary and the time() just after
+const utcoffset = floor(Dates.datetime2unix(now())) - floor(time())
+
+function get_log_ts(precision=6)
+    t = time()
+    ms = round(t - floor(t), precision)
+    dt = Dates.unix2datetime(floor(t+utcoffset))
+    msstr = rpad(string(ms)[3:end], precision, "0")
+    ts = "$(string(dt)[12:end]).$msstr"
+end
+
+function get_log_preface()
+    ts = get_log_ts()
+    taskname = get(task_local_storage(), :IJulia_task, "")
+    "$ts($taskname): "
+end
```

Comment:
If we want accurate times on the logs, then yes :) I'm personally pro the accurate times, they were useful to me here, and may be useful down the track. However, I'm not going to throw a wobbly delete the whole pull request if you feel differently :) happy to accept whatever you decide. Btw in my other branch I moved these and `vprintln`,`verror_show` to a separate debug.jl file, which may be more palatable?